packages/lib-react-components/src/ZooHeader/ZooHeader.js

```diff
@@ -55,12 +57,14 @@ export default function ZooHeader (props) {
   return (
     <StyledHeader
       background='black'
+      className={className}
```

Comment:
You shouldn't need `className` if you're already spreading `props`

Reply:
Yes, true. I was a bit at my wits end yesterday going around adding these props everywhere.